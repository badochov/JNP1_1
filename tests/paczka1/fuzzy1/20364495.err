Error in line 2: b9eE0 A190 62,9
Error in line 3: b9eE0 A24 26,0
Error in line 6: BDEJUdH A66 37,3
Error in line 7: b9eE0 A190 93,0
Error in line 12: b9eE0 A66 69,7
Error in line 9: meIFtae A190 26,9
Error in line 23: meIFtae S471 49,1
Error in line 24: meIFtae A24 14,9
Error in line 29: meIFtae S471 22,9
Error in line 32: meIFtae A24 99,4
