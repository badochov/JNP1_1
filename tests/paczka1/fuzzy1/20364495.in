S471 A190 49,5
b9eE0 A190 62,9
b9eE0 A24 26,0
orVNhjbmA A66 26,5
?S471
BDEJUdH A66 37,3
b9eE0 A190 93,0
BDEJUdH S471 64,0
meIFtae A190 26,9
S471 A190 236,1

b9eE0 A66 69,7
b9eE0 A24 24,5
orVNhjbmA A66 46,7
1jL A24 9,2
?S471
orVNhjbmA A66 33,6
?
meIFtae A24 50,5
meIFtae A24 9,1
1jL A24 5,3
BDEJUdH S471 218,9
meIFtae S471 49,1
meIFtae A24 14,9
b9eE0 A24 81,8
BDEJUdH A24 99,9
?A24
BDEJUdH A24 132,9
meIFtae S471 22,9

f0MeqAq A190 21,6
meIFtae A24 99,4
?BDEJUdH
?b9eE0
f0MeqAq A190 21,2
?S471
f0MeqAq A66 87,5
S471 S471 58,0
meIFtae A190 77,8
f0MeqAq A66 26,3