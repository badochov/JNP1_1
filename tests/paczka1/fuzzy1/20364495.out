S471 A 186,6
S471 A 186,6
orVNhjbmA A 20,2
A66 20,2
A190 186,6
A24 102,6
BDEJUdH A 33,0 S 154,9
b9eE0 A 57,3
S471 A 186,6
S471 154,9
